### DISCUSSION TITLE
Find method or findAll method does not work properly in Windows

### DISCUSSION THREAD

**Discussion by foreverabhi2002** [mikro-orm/mikro-orm#4894] [General]
**Describe the bug**
Find() method or findAll() method does not work properly in Windows, however, for Mac and Linux it's working fine. When executing the API method for the first time it outputs the correct response but after that, if hit the API for the rest of the time it prints 1st relation data for the complete response. I faced this issue when I was using MongoDB for a database and was not sure about other databases.

**To Reproduce**
1. First implement the find method in a Windows-based system and MongoDB as a database.

**Expected behavior**
![image](https://github.com/mikro-orm/mikro-orm/assets/57245064/b14cb363-cb13-4416-a4e6-8d82573756f7)

**Output behavior**
![image_2023-11-03_18-01-13](https://github.com/mikro-orm/mikro-orm/assets/57245064/467b3663-498d-469e-ad0b-220de0917690)

**Versions**

| Dependency | Version |
| - | - |
| node | 20 | 
| typescript | 6 |
| mikro-orm | 5.8.3 |
| mongodb | 6.1 |

**Comment by B4nan** [comment]
> First implement the find method in a Windows-based system and MongoDB as a database.

This is not a reproduction. Please provide a complete reproduction, a code I can run myself. It should be minimal, no web servers ideally. The issue you are describing is not about `findAll` method itself, it's about a context mismatch, it's about how you work with it. And often about other parts of your toolchain, not about the ORM itself.

I also didn't really understand what you are trying to say with the screenshots, without seeing the code that produces the logs, it only adds confusion.
